src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts

```diff
@@ -238,46 +218,13 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
   }
 
   onAccessPermissions(): void {
-    if (!this.group || !this.progressOverlay) return;
-    const { title, groupId: targetGroupId, itemId } = this.progressOverlay.accessPermissions;
-
     this.hideProgressDetail();
-    this.dialogTitle = title;
-    this.dialog = 'loading';
-
-    this.permissionsFetchingSubscription?.unsubscribe();
-    this.permissionsFetchingSubscription = this.groupPermissionsService.getPermissions(this.group.id, targetGroupId, itemId)
-      .subscribe({
-        next: permissions => {
-          this.dialogPermissions = {
-            itemId: itemId,
-            targetGroupId: targetGroupId,
-            permissions: permissions.granted,
-          };
-          this.dialog = 'opened';
-        },
-        error: () =>
-          this.actionFeedbackService.error($localize`The permissions cannot be retrieved. ` +
-            $localize`:@@contactUs:If the problem persists, please contact us.`),
-      });
+    this.dialog = 'opened';
   }
 
   onDialogClose(): void {
     this.dialog = 'closed';
```

Comment:
Does it still make sense not just to have boolean ?

Reply:
Fixed